src/components/complete-profile-second/CompleteProfileSecond.tsx

```diff
@@ -1,20 +1,39 @@
-import { memo, useState } from 'react';
+import { memo, useState, useEffect } from 'react';
+
 import { useLocales } from 'src/locales';
 import WorkForm from 'src/components/complete-profile-second/work-form/WorkForm';
 import WorkList from 'src/components/complete-profile-second/work-list/WorkList';
+import { profileApi } from 'src/redux/api/profileCompleteApi';
```

Comment:
It's ok, but let's have the same structure everywhere. Mutations which you want to get from profileApi are exported separately in redux, no need to import api